src/Components/Facility/Consultations/LiveFeed.tsx

```diff
@@ -251,20 +248,22 @@ const LiveFeed = (props: any) => {
           if (currentPreset?.asset_object?.id && data?.position) {
             setLoading(option.loadingLabel);
             console.log("Updating Preset");
-            const response = await dispatch(
-              partialUpdateAssetBed(
-                {
+            const { res: responseRes } = await request(
```

Comment:
Why is the variable named `responseRes` when you can keep it `res` itself?

```suggestion
            const { res } = await request(
```